hu.irl.cameractrls.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>hu.irl.cameractrls</id>
+    <launchable type="desktop-id">hu.irl.cameractrls.desktop</launchable>
+    <name>cameractrls</name>
+    <summary>Camera controls for Linux</summary>
+    <description>
+        <p>
+            It's a standalone Python CLI and GUI (GTK, TK) and camera Viewer (SDL) to set the camera controls in Linux.
+            It can set the V4L2 controls and it is extendable with the non standard controls.
+            Currently it has a Logitech extension (Led mode, led frequency), Kiyo Pro extension (HDR, HDR mode, FoV, AF mode, Save),
+            Systemd extension (Save and restore controls with Systemd path+service if available).
```

Comment:
How does it actually work from Flatpak?

Reply:
It works well. The save/restore functionality is not active, because it detects there is no systemd.

Reply:
The point of my question is, does it make sense to mention systemd at all, if the function is not available?

Reply:
Removed from the description.